model/src/cmd/ModelCommandAPI.cc

```diff
@@ -0,0 +1,369 @@
+                  "      [" + std::to_string(poseComp->Data().Roll()) + " | "
+                            + std::to_string(poseComp->Data().Pitch()) + " | "
+                            + std::to_string(poseComp->Data().Yaw()) + "]";
+
+    std::cout << "Name: " << nameComp->Data() << std::endl
+              << "  - Pose: " << poseInfo << std::endl << std::endl;
+  }
+}
+
+//////////////////////////////////////////////////
+// \brief Print the model links information.
+// \param[in] _entity Entity of the model requested.
+// \param[in] _ecm ECM ready for requests.
+// \param[in] _linkName Link to be printed, if nullptr, print all links.
+void printLinks(const uint64_t _entity,
+                const ignition::gazebo::EntityComponentManager &_ecm,
+                const std::string &_linkName)
+{
+  const auto links = _ecm.EntitiesByComponents(
+  ignition::gazebo::components::ParentEntity(_entity),
+  ignition::gazebo::components::Link());
+  for(const auto &entity : links)
+  {
+    const auto parentComp =
+        _ecm.Component<ignition::gazebo::components::ParentEntity>(entity);
+
+    const auto nameComp =
+        _ecm.Component<ignition::gazebo::components::Name>(entity);
+
+    if(_linkName.length() && _linkName != nameComp->Data())
+        continue;
+
+    if (parentComp)
+    {
+      std::string parentInfo;
+      const auto parentNameComp =
+          _ecm.Component<ignition::gazebo::components::Name>(
+          parentComp->Data());
+
+      if (parentNameComp)
+      {
+        parentInfo += parentNameComp->Data() + " ";
+      }
+      parentInfo += "[" + std::to_string(parentComp->Data()) + "]";
+      std::cout << "  - Link [" << entity << "]" << std::endl
+                << "    - Name: " << nameComp->Data() << std::endl
+                << "    - Parent: " << parentInfo << std::endl;
+    }
+
+    const auto inertialComp =
+        _ecm.Component<ignition::gazebo::components::Inertial>(entity);
+
+    if (inertialComp)
+    {
+      const auto inertialMatrix =  inertialComp->Data().MassMatrix();
+      const auto massComp = inertialComp->Data().MassMatrix().Mass();
+
+      const std::string massInfo = "[" + std::to_string(massComp) + "]";
+      const std::string inertialInfo =
+            "\n        [" + std::to_string(inertialMatrix.Ixx()) + " | "
+                          + std::to_string(inertialMatrix.Ixy()) + " | "
+                          + std::to_string(inertialMatrix.Ixz()) + "]\n"
+              "        [" + std::to_string(inertialMatrix.Ixy()) + " | "
+                          + std::to_string(inertialMatrix.Iyy()) + " | "
+                          + std::to_string(inertialMatrix.Iyz()) + "]\n"
+              "        [" + std::to_string(inertialMatrix.Ixz()) + " | "
+                          + std::to_string(inertialMatrix.Iyz()) + " | "
+                          + std::to_string(inertialMatrix.Izz()) + "]";
+      std::cout << "    - Mass: " << massInfo << std::endl
```

Comment:
What do you think of printing the units for all quantities? This is something users often get confused about. Here's one idea of how it could look like:

```
Name: box
  - Pose [ XYZ (m) ] [ RPY (rad) ]: 
      [0.000000 | 0.000000 | 0.500000]
      [0.000000 | -0.000000 | 0.000000]

  - Link [9]
    - Name: box_link
    - Parent: box [8]
    - Mass (kg): [1.000000]
    - Inertial Matrix (kg⋅m^2): 
        [1.000000 | 0.000000 | 0.000000]
        [0.000000 | 1.000000 | 0.000000]
        [0.000000 | 0.000000 | 1.000000]
    - Pose [ XYZ (m) ] [ RPY (rad) ]: 
        [0.000000 | 0.000000 | 0.000000]
        [0.000000 | -0.000000 | 0.000000]
```

Reply:
Addressed